packages/wordpress-kit/src/lib/tailwindcssPlugin.ts

```diff
@@ -174,6 +174,74 @@ export default plugin(function ({ addUtilities, theme }) {
     {}
   );
 
+  const tableComponent = {
+    '.wp-block-table': {
+      td: {
+        padding: '0.5em',
+      },
+      table: {
+        thead: {
+          tr: {
+            borderColor: 'inherit',
+            th: {
+              padding: '0.5em',
+              color: 'inherit',
+            },
+          },
+          borderColor: 'inherit',
+        },
+        tbody: {
+          borderColor: 'inherit',
+          tr: {
+            borderColor: 'inherit',
+          },
+        },
+        '&.has-fixed-layout': {
+          width: 'unset',
+          tableLayout: 'fixed',
+        },
+        maxWidth: '650px',
+        margin: 'auto',
+      },
+      figcaption: {
+        fontSize: '.9rem',
+        textAlign: 'center',
+      },
+      '&.is-style-stripes': {
+        margin: '0',
+        table: {
+          tbody: {
+            'tr:nth-child(odd)': {
+              backgroundColor: theme('colors.stripes', '#f2f2f2'),
+            },
+          },
+        },
+      },
+      '&.alignwide': {
+        padding: '0 1.5rem',
+        table: {
+          maxWidth: '850px',
+        },
+      },
+      '&.alignfull': {
+        [`@media (min-width:${theme(
+          'screen.xl',
+          '1280px'
+        )})` as '@media(min-width:1280px)`']: {
+          marginLeft: 'calc(-1 * max(1rem, 10vw))',
+          marginRight: 'calc(-1 * max(1rem, 10vw))',
```

Comment:
This is some nice wizardry. Can you add a comment or two explaining what the `calc` is doing, and also maybe the what is going on with the type coercion so we can more easily maintain this in the future?